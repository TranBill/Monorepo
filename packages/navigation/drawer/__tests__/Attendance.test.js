import React from 'react';
import AttendanceScreen from '../src/screens/Attendance';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<AttendanceScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});