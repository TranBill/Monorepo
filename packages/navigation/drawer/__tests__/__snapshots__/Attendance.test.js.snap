// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#808080",
        "fontSize": 25,
        "textAlign": "auto",
      }
    }
  >
    This is attendance
  </Text>
</View>
`;
